### PR TITLE
Aidinzp

### PR DESCRIPTION
**Pull request by aidinzp**
Hi there, 

This is my first pull request for the website profile!

Thank you!

### CODE REVIEW

File: content/authors/aidinzp/_index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  icon_pack: fab
+ link: https://www.linkedin.com/in/aidinzp/
+
+user_groups:
+- committee: members
+  role: Team 4 
+
+tags:
+- '2021'
+Interested in various fields of Mathematics and Data Science, specifically Machine Learning, Convolutional Neural Networks, as well as Partial Differential Equations. Have been working on some projects for the past few months.
```
Comment (ianabc) [inline content/authors/aidinzp/_index.md:50]: I think the stuff on this line should be moved to the end of the file (after the line which says `---`). The area after `---` supports markdown.

File: content/authors/aidinzp/_index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+social:
+- icon: github
+  icon_pack: fab
+  link: https://github.com/aidinzp
+#- icon: twitter
+#  icon_pack: fab
+#  link: https://twitter.com/carlosshigoto
+- icon: linkedin
+  icon_pack: fab
+ link: https://www.linkedin.com/in/aidinzp/
```
Comment (ianabc) [inline content/authors/aidinzp/_index.md:42]: I think there is a space missing from line 42

**Review by ianabc — CHANGES_REQUESTED** [review]
HI Aidin,

I think this is 90% of the way there just two minor points. One is a whitespace issue and one is a little thing with the markdown at the end of the file (you need to put the markdown _after_ the `---` line).

**Comment by aidinzp** [comment]
Hi Ian, 

I have made some changes to that. Hopefully it works now. Thank you!

**Pull request merged**